setup.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+# -- coding: utf-8 --
+# Copyright 2017 Reactive Ops Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import glob
+from setuptools import setup, find_packages
+
+try:
+    from setuptools import setup, find_packages
+except ImportError:
+    print("setup tools required. Please run: "
+          "pip install setuptools).")
+    sys.exit(1)
+
+__version__ = '4.1.1'
```

Comment:
How do we keep this in sync with the package.json. Do we even care to? Can we set this when do the release to pypi so that it matches that tag that we're releasing?

Reply:
We can update the setup.py to pull the version from the package.json easily enough but that means we have to sync the version from the tag with the version in the package.json. And we can send this to pypi.. 

But mostly we are using git to install rok8s scripts and even in the tags, the version is set to 4.1.0. Don't most packages put the version in source control somewhere so it can be installed from the remote repo (npm/pypi) OR the git repo and get the same version number?

I realize that makes tagging a two step process, committing the new version number and then tagging. Thoughts?

Reply:
I'm pretty un-opinionated about how we manage the version. Since 7.2.0 is published, let's change these to 7.2.1 and then tag that.